.yarnrc

```diff
@@ -2,5 +2,4 @@
 # Issue: https://github.com/yarnpkg/yarn/issues/5891
 --registry "https://registry.npmjs.org/"
 --scripts-prepend-node-path true
---cache-folder "~/.cache"
```

Comment:
What has this got to do with the PR?

Reply:
it turns out this flag breaks everything
As a result, a `store` directory appeared in my directory `./store/~/.cache`
Yarn doesn't create cache directories correctly

Reply:
ok